tests/console/yast2_apparmor.pm

```diff
@@ -43,8 +43,10 @@ sub run {
     } else {
         #SLES >=15 imediatelly asks for extra packages, not after main menu:
         install_extra_packages_requested;
-        send_key 'alt-l';
         assert_screen 'yast2_apparmor';
+        send_key 'alt-l';
+        #SLE 15+ takes some extra time to load next screen:
+        wait_still_screen 5;
```

Comment:
I guess we can then bump timeout for the `assert_screen` on L52.

Reply:
true, I did a better distributions of the timeouts, lets see how the more loaded main instance likes that.

thanks